lib/engine/step/g_18_cz/buy_sell_par_shares.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative '../buy_sell_par_shares.rb'
+
+module Engine
```

Comment:
what is this for

Reply:
I copied too much. I removed the require statement